src/components/shell/shell.js

```diff
@@ -265,7 +267,9 @@ class Shell extends BaseComponent {
    * @memberof Shell
    */
   render(props: any): React$Element<any> {
-    let playerClasses = [style.player, style.skinDefault];
+    let playerClasses = [style.player, style.skinDefault,
+      __CSS_MODULE_PREFIX__ + '-' + this.player.env.os.name.replace(/ /g, "_"),
+      __CSS_MODULE_PREFIX__ + '-' + this.player.env.browser.name.replace(/ /g, "_")];
```

Comment:
No need to re calculate it on every time render has been called. We can calculate it one time in the constructor and use it here.
```js
this._osClass = `${__CSS_MODULE_PREFIX__}-${this.player.env.os.name.replace(/ /g, '-')}`;
this._browserClass = `${__CSS_MODULE_PREFIX__}-${this.player.env.browser.name.replace(/ /g, '-')}`;

```
One more thing is to change the replacer string to '-' instead of '_' since this is our convention in css class names.

Reply:
done